tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@react-native/typescript-config/tsconfig.json",
+  "compilerOptions": {
+    "types": ["react-native"],
```

Comment:
is this necessary? i've never had to do this 🤔

Reply:
I overwrited `types` from `@react-native/typescript-config` 
to omit jest types as this library doesn't have any jest tests